plugins/restlet1x/nexus-restlet1x-plugin/src/main/java/org/sonatype/nexus/rest/repositorystatuses/RepositoryStatusesListPlexusResource.java

```diff
@@ -76,6 +84,15 @@ public PathProtectionDescriptor getResourceProtection() {
   public Object get(Context context, Request request, Response response, Variant variant)
       throws ResourceException
   {
+    if (adminOnly) {
+      Subject subject = ThreadContext.getSubject();
+      // skip building reply unless user has admin access (ie. can update status)
+      if (subject == null || !subject.isPermitted("nexus:repostatus:update")) {
```

Comment:
@mcculls understood, but we should ensure that user knows this as we asked them something different, which they responded with:

"HSBC indicates they would accept a solution where the respository_statuses resource would be a no-op for non Nexus Administrator role users"

So if they have customized their security to give non-admin users that permission, it may still behave poorly for them.